jobs.go

```diff
@@ -376,6 +377,18 @@ func (s *Job) UsesVolumes() bool {
 	return false
 }
 
+// Returns a list of inputs that do not have download tickets.
+func (job *Job) FilterInputsNoTickets() []StepInput {
```

Comment:
I don't know if this name is consistent with other stuff in this library, but part of me feels like it should be `Without` rather than `No` to match the `With` function below.

Obviously not, uh, super important though

Reply:
I thought it would be slightly faster to tell the difference between `FilterInputsNoTickets` and `FilterInputsWithTickets`, but now I'm not seeing much of a difference:
`FilterInputsNoTickets`
`FilterInputsWithTickets`

`FilterInputsWithoutTickets`
`FilterInputsWithTickets`

`FilterInputsWithoutTickets` makes a bit more sense, so I'll update it 👍